src/main/java/io/github/ygojson/application/core/db/set/SetEntity.java

```diff
@@ -0,0 +1,474 @@
+package io.github.ygojson.application.core.db.set;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.function.BiConsumer;
+
+import jakarta.persistence.*;
+
+@Entity
+@Table(name = "SET_TABLE")
+public class SetEntity {
+
+	// runtime DB ID
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE)
+	private Long id;
+
+	// YGOJSON-ID
+	private UUID ygojsonId;
+	// common independently of the localization
+	private String printNumberPrefix;
+	private String type;
+	private String series;
+
+	// name (localized)
+	private String nameEn;
```

Comment:
Change the order to be `enName` (and same for the rest of localized properties).